pkg/methods/irma_validator_test.go

```diff
@@ -118,14 +118,14 @@ func TestValidateContract(t *testing.T) {
 			"an expired contract should be invalid",
 			args{
 				base64.StdEncoding.EncodeToString([]byte(testdata.ValidIrmaContract)),
-				types2.IrmaFormat,
+				types.IrmaFormat,
 				"Helder",
```

Comment:
I wouldn't use a Nedap product name in Nuts reference implementation source code

Reply:
Creating a new contract is a bit of a hassle. This type of contract is going to disapear with the introduction of the new OAuth flow.